packages/contracts/deploy/018-fund-accounts.ts

```diff
@@ -0,0 +1,37 @@
+/* Imports: External */
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+
+/* Imports: Internal */
+import { getDeployedContract } from '../src/hardhat-deploy-ethers'
+import { ethers } from 'hardhat'
+
+const deployFn: DeployFunction = async (hre) => {
+  const { deployer } = await hre.getNamedAccounts()
+
+   // Fund all default hardhat signers  with ETH if deploying on a local hardhat network
+   const { chainId } = await ethers.provider.getNetwork()
+   if (chainId === 31337) {
+     const Proxy__OVM_L1ETHGateway = await getDeployedContract(
+       hre,
+       'Proxy__OVM_L1ETHGateway',
+       {
+         signerOrProvider: deployer,
+         iface: 'OVM_L1ETHGateway',
+       }
+     )
+     const signers = await ethers.getSigners()
+     for (const signer of signers) {
```

Comment:
This should be a part of `hardhat-ovm` and should not be part of the deployment process. We should also do this funding in parallel because doing in serial takes a LONG time.

Reply:
Dropping the commit from this PR, it's a separate feature anyways.